Contributing.md

```diff
@@ -0,0 +1,79 @@
+# Contributing
+There are many ways to contribute to this project such as testing, design, and development.
+
+## Development
+This section provides information for people who are interested in contributing code.
+
+### Setup
+Fork this repository and clone it to your local development environment. 
+In the root folder (`cc-global-components`), run the following commands:
+```npm
+npm install
+```
+To preview the project, run
+```npm
+npm run serve
+```
+To build the project, run
+```npm
+npm run build
+```
+To build the project just for CDN use, run
+```npm
+npm run build:unpkg
+```
+
+### Testing Locally
+To test locally how it will behave when hosted on a CDN, create a base HTML file: 
+  ```html
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Testing CC Global Components</title>
+    </head>
+    <body>
+        <div id="explore-cc">
+            <cc-explore />
+        </div>
+        <div id="header-cc">
+            <cc-global-header />
+        </div>
+        <div id="footer-cc">
+            <cc-global-footer />
+        </div>
+        <script src="https://unpkg.com/vue@next"></script>
+        <script src="./dist/cc-global.min.js"></script>
+        <script>
+            const cc_explore = Vue.createApp({})
+            cc_explore.use(CcGlobal)
+            cc_explore.mount('#explore-cc')
+
+            const cc_header = Vue.createApp({})
+            cc_header.use(CcGlobal)
+            cc_header.mount('#header-cc')
+
+            const cc_footer = Vue.createApp({})
+            cc_footer.use(CcGlobal)
+            cc_footer.mount('#footer-cc')
+            
+        </script>
+    </body>
+    </html>
```

Comment:
Let's just include this HTML file, e.g. as `dev.html`, `test.html`, or something similar.